src/development/tools/devtools/performance.md

```diff
@@ -113,18 +135,62 @@ red:
 check out [Reduce shader compilation jank on mobile][].
 
-## Timeline events chart
+## Frame analysis tab
+
+Selecting a janky frame (slow, colored in red)
+from the Flutter frames chart above shows debugging hints
+in the Frame analysis tab. These hints help you diagnose
+jank in your app, and notify you of any expensive operations
+that we have detected that might have contributed to the slow frame time.
+
+![Screenshot of the frame analysis tab]({{site.url}}/assets/images/docs/tools/devtools/frame-analysis-tab.png)
+
+## Raster stats tab
+
+{{site.alert.note}}
+  For best results, this tool should be used with
+  the Impeller rendering engine. When using Skia,
+  the raster stats reported might be inconsistent
+  due to the timing of when shaders are compiled.
+{{site.alert.end}}
+
+If you have Flutter frames that are janking with
+slow raster thread times, this tool might be able
+to help you diagnose the source of the slow performance.
+To generate raster stats:
+
+1. Navigate to the screen in your app where you are seeing
+   raster thread jank.
+2. Click **Take Snapshot**.
+3. View different layers and their respective rendering times.
+
+If you see an expensive layer, find the Dart code in your app
+that is producing this layer and investigate further.
+You can make changes to your code, hot reload,
+and take new snapshots to see if the performance of a layer
+was improved by your change.
+
+![Screenshot of the raster stats tab]({{site.url}}/assets/images/docs/tools/devtools/raster-stats-tab.png)
+
+## Timeline events tab
 
 The timeline events chart shows all event tracing from your application.
 The Flutter framework emits timeline events as it works to build frames,
-draw scenes, and track other activity such as HTTP traffic.
-These events show up here in the Timeline.
+draw scenes, and track other activity such as HTTP request timings
+and garbage collection. These events show up here in the Timeline.
 You can also send your own Timeline events using the dart:developer
-[`Timeline`]({{site.api}}/flutter/dart-developer/Timeline-class.html)
-and [`TimelineTask`]({{site.api}}/flutter/dart-developer/TimelineTask-class.html)
-APIs.
+[`Timeline`][] and [`TimelineTask`][] APIs.
+
+[`Timeline`]: {{site.api}}/flutter/dart-developer/Timeline-class.html
+[`TimelineTask`]: {{site.api}}/flutter/dart-developer/TimelineTask-class.html
+
+![Screenshot of a timeline events tab]({{site.url}}/assets/images/docs/tools/devtools/timeline-events-tab.png)
+For help with navigating and using the trace viewer,
+click the **?** button at the top right of the timeline
+events tab bar. To refresh the timeline with new events from
+your application, click the refresh button
+(also in the upper right corner of the tab controls).
 
-![Screenshot of timeline events for a frame]({{site.url}}/assets/images/docs/tools/devtools/performance-timeline-events-chart.png){:width="100%"}
 
 The flame chart supports zooming and panning:
```

Comment:
from here to ## ADvanced debugging tools should be removed. It no longer applies after switching to the Perfetto trace viewer

Reply:
Looks like too much was deleted here. We still need the section on shader compilation and the timeline events chart. My comment was referring to the section from this line down to `## Advanced debugging`:

```
The flame chart supports zooming and panning:

* To zoom, scroll up and down with the mouse wheel / trackpad
* To pan horizontally, either click and drag the chart or
  scroll horizontally with the mouse wheel / trackpad
* To pan vertically, either click and drag the chart or use
  **alt + scroll**
* The WASD keys also work for controlling zoom and horizontal scroll position

You can click an event to view CPU profiling information in the CPU profiler
below, described in the next section.
```

@sfshaza2 Can you add these sections back with the new updates? (basically what you had prior to your last commit, minus the section I described in the previous paragraph)